### PR TITLE
Add UserApi.getMemberships() to list projects and groups a user is member of

### PR DESCRIPTION
**Pull request by bwaldvogel**

### CODE REVIEW

File: src/main/java/org/gitlab4j/api/models/MembershipSourceType.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.gitlab4j.api.models;
+
+public enum MembershipSourceType {
```
Comment (jabby) [inline src/main/java/org/gitlab4j/api/models/MembershipSourceType.java:3]: In this project all enum names should be capitalized. Can you modify this enum like [PackageType](https://github.com/gitlab4j/gitlab4j-api/blob/master/src/main/java/org/gitlab4j/api/models/PackageType.java)

**Review by jabby — CHANGES_REQUESTED** [review]
Thanks for your contribution. It needs a little modification. Can you take care of this one?

File: src/main/java/org/gitlab4j/api/models/MembershipSourceType.java (hunk 1 above)
Comment (bwaldvogel) [reply src/main/java/org/gitlab4j/api/models/MembershipSourceType.java:8]: Oh, sorry. Didn’t notice this coding convention.
I’ve adjusted the PR as you suggested.

Comment (jabby) [reply src/main/java/org/gitlab4j/api/models/MembershipSourceType.java:8]: Thanks

**Review by jabby — APPROVED** [review]

**Comment by bwaldvogel** [comment]
I rebased my PR in the hope that the tests will succeed now. (Locally it succeeded)

**Pull request merged**